test/stdlib/TestCharacterSet.swift

```diff
@@ -192,6 +192,22 @@ class TestCharacterSet : TestCharacterSetSuper {
         expectEqual(expected, union)
     }
 
+    func test_subtractAndFormSymmetricDifference() {
+        var set1 = CharacterSet(charactersIn: "abc")
+        let set2 = CharacterSet(charactersIn: "b")
+        set1.subtract(set2)
+        expectFalse(set1.contains("b"))
+        set1.formSymmetricDifference(set2)
+        expectTrue(set1.contains("b"))
+
+        let expected = set1
+        var set3 = CharacterSet()
```

Comment:
We're not too strict on "one assert per test", but if it helps clear up what's going on then that's fine.